smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftTypes.kt

```diff
@@ -16,6 +16,8 @@ object SwiftTypes {
     val Double = builtInSymbol("Double")
     val Bool = builtInSymbol("Bool")
 
+    val StringArray = builtInSymbol("Array<Swift.String>")
```

Comment:
Might be good idea to make this `[String]` to make it idiomatic

Reply:
This form is needed because when the SwiftWriter prints this symbol, it will preface it with the namespace:
```
Swift.Array<Swift.String>
```
Awkward but it compiles.  On the other hand:
```
Swift.[Swift.String]
```
does not compile: Swift doesn't allow tacking a namespace on when you declare an Array type with `[...]` syntax.